lib/generators/decidim/templates/initializer.rb

```diff
@@ -6,8 +6,8 @@
   config.mailer_sender = "change-me@domain.org"
   config.authorization_handlers = [ExampleAuthorizationHandler]
 
-  # Uncomment this lines to set your preferred locales
-  # config.available_locales = [:en, :ca, :es]
+  # Change this line to set your preferred locales
+  config.available_locales = [:en, :ca, :es]
```

Comment:
Maybe the problems are the seeds, then? Since the organization created there uses `Decidim.available_locales` as its own `available_locales`. Maybe we can change it there?